gulpfile.js

```diff
@@ -0,0 +1,306 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const fs = require('fs');
+const nls = require('vscode-nls-dev');
+const path = require('path');
+const minimist = require('minimist');
+const es = require('event-stream');
+const sourcemaps = require('gulp-sourcemaps');
+const ts = require('gulp-typescript');
+const typescript = require('typescript');
+const tsProject = ts.createProject('./tsconfig.json', { typescript });
+const filter = require('gulp-filter');
+const vinyl = require('vinyl');
+const jsonc = require('jsonc-parser');
+
+
+// Patterns to find schema files
```

Comment:
If you don't have any JSON schema files, you could remove the code in here that translates those.

Reply:
We will have in future. We'll spend less time removing now and adding back then. :).